runtime/sema/authaccount_contracts.go

```diff
@@ -193,6 +195,7 @@ Returns nil if no contract/contract interface with the given name exists in the
 `
 
 var AuthAccountContractsTypeRemoveFunctionType = &FunctionType{
+	Purity: FunctionPurityImpure,
```

Comment:
Doesn't have to be in this PR, but maybe we could replace the struct literals with a constructor to enforce the proper purity status